serverless/packages/functions/src/trainspace/trainspace-data.ts

```diff
@@ -0,0 +1,79 @@
+import { TrainStatus } from './constants';
+import { PutItemCommandInput } from "@aws-sdk/client-dynamodb";
+
+export default class TrainspaceData
+{
+    trainspace_id: string;
+    uid: string;
+    created: string = "";
+    data_source: string = "";
+    dataset_data: object = {};
+    name: string = "";
+    parameters_data: object = {};
+    review_data: string = "";
+    status: TrainStatus = TrainStatus.QUEUED;
+
+    constructor(trainspace_id: string, uid: string, data_source: string, dataset_data: object, name: string, parameters_data: object, review_data: string) {
+        this.trainspace_id = trainspace_id;
+        this.uid = uid;
+        this.created = this.getDateString();
+        this.data_source = data_source;
+        this.dataset_data = dataset_data;
+        this.name = name;
+        this.parameters_data = parameters_data;
+        this.review_data = review_data;
+    }
+
+    getDateString() : string {
+        return Date.now().toString();
+    }
+
+    convertToDynamoItemInput(tableName: string) : PutItemCommandInput {

```

Comment:
I don't think a class is necessary for this, since the convertToDynamoItemInput can be generalized so that it applies to any dynamo item. What's left of the class would then just be the data, which would defeat the purpose of having the class in the first place